.github/release.yml

```diff
@@ -0,0 +1,17 @@
+# Release template for GitHub Releases
+
+changelog:
+  categories:
+    - title: ✨ Features
+      labels:
+        - feature
+    - title: 🐛 Bugfixes
+      labels:
+        - bug-fix
+    - title: ✏️ Tasks
+      labels:
+        - '*'
+      exclude:
+        labels:
+          - feature
+          - bug-fix
```

Comment:
Added labels for these, https://github.com/ComposeGears/Valkyrie/labels

Reply:
Labeled https://github.com/ComposeGears/Valkyrie/releases/tag/Nightly to https://github.com/ComposeGears/Valkyrie/pulls?q=is:pr+label:feature+is:closed

Reply:
Nice feature! 
Do I understand correctly that all tasks should be covered by the 'housekeeping' label?"

Reply:
`✏️ Tasks` now contains all but `feature` and `bug-fix` (`housekeeping`, `dependencies` and other categories).

Reply:
Ok, let's process with this, I like automation 🌚

Reply:
Or we can migrate to using [keep-a-changelog](https://github.com/olivierlacan/keep-a-changelog) and auto-release like https://github.com/olivierlacan/keep-a-changelog/pull/629.

Reply:
Changelog generation into file can be useful for integration with this plugin feature: https://github.com/JetBrains/gradle-changelog-plugin

It allows to see changelog in Jetbrains marketplace

Reply:
Yep.

Reply:
@Goooler 
Checked generated report, looks pretty nice, thanks for contribution ❤️ 

I see few things to improve:
- can we add "🆙 Dependencies" section based on label (for easy track changes)?
- is it possible to change title from "Nightly" to smth like "0.0.6-Nightly"?

<img width="389" alt="image" src="https://github.com/user-attachments/assets/b9749a37-1079-4f57-9d3d-0e613090c5c8">

Reply:
1. Yes we can.
2. We can keep the version name like 0.0.6-SNAPSHOT.

Reply:
yes, perfect! 
This is not urgent at all 🙂